Package.swift

```diff
@@ -2,6 +2,11 @@ import PackageDescription
 
 let package = Package(
     name: "SwiftScript",
+    targets: [
+        Target(name: "SwiftAST"),
+        Target(name: "SwiftParse", dependencies: ["SwiftAST"]),
+        Target(name: "SwiftScript", dependencies: ["SwiftParse"]),
```

Comment:
I suggest to add `"SwiftAST"` to`SwiftScript` dependencies as well.
Although it's not necessary, sources in `SwiftScript` directly import `SwiftAST`.